roles/openshift_sanitize_inventory/tasks/deprecations.yml

```diff
@@ -18,8 +18,5 @@
   when:
   - dep_check.changed
 
-# for with_fileglob Ansible resolves the path relative to the roles/<rolename>/files directory
 - name: Assign deprecated variables to correct counterparts
-  include_tasks: "{{ item }}"
```

Comment:
including task files this way doesn't seem really necessary; we have to create the task files, might as well add one line of ansible if needed.

Reply:
:+1: the original thought here was to create scaffolding that other groups could use as well.